src/main/resources/swagger/agora.yaml

```diff
@@ -1406,6 +1406,52 @@ paths:
             - openid
             - email
             - profile
+  /ga4gh/tools/{id}/versions/{version-id}/{type}/descriptor:
+    get:
+      summary: Get the tool descriptor (CWL/WDL) for the specified tool.
+      description: Returns the CWL or WDL descriptor for the specified tool.
+      tags:
+        - GA4GH
+      parameters:
+        - name: type
+          required: true
+          in: path
+          description: >
+            The output type of the descriptor. If not specified it is up to the underlying
+            implementation to determine which output type to return. Plain types return the
+            bare descriptor while the "non-plain" types return a descriptor wrapped
+            with metadata.
+            In Agora, only WDL and plain-WDL are supported.
+          type: string
+          enum:
+            - CWL
+            - WDL
+            - plain-CWL
+            - plain-WDL
+        - name: id
+          in: path
+          description: >
+            A unique identifier of the tool, scoped to this registry, for example `123456`.
+            In Agora, this must be a namespace + ":" + name. For instance, if your namespace
+            is 'foo' and name is 'bar', this must be 'foo:bar'.
+          required: true
+          type: string
+        - name: version-id
+          in: path
+          required: true
+          type: string
+          description: >
+            An identifier of the tool version for this particular tool registry, for example `v1`.
+            In Agora, this must be an integer representing the Agora snapshot id.
+      responses:
+        '200':
+          description: The tool descriptor.
+          schema:
+            $ref: '#/definitions/ToolDescriptor'
+        '404':
+          description: The tool can not be output in the specified type.
+          schema:
+            $ref: '#/definitions/Error'
```

Comment:
I kept the swagger definitions as close as possible to spec: https://github.com/ga4gh/tool-registry-schemas/blob/develop/src/main/resources/swagger/ga4gh-tool-discovery.yaml. I just added a few "In Agora ... " lines of description.